packages/bsky/src/api/app/bsky/feed/getLikes.ts

```diff
@@ -68,18 +72,40 @@ const hydration = async (inputs: {
   skeleton: Skeleton
 }) => {
   const { ctx, params, skeleton } = inputs
-  return await ctx.hydrator.hydrateLikes(skeleton.likes, params.hydrateCtx)
+  const likesState = await ctx.hydrator.hydrateLikes(
+    skeleton.authorDid,
+    skeleton.likes,
+    params.hydrateCtx,
+  )
+  const dids = [skeleton.authorDid]
+
+  if (likesState.likes) {
+    for (const like of likesState.likes.values()) {
+      if (like) {
+        dids.push(like.cid)
```

Comment:
The `like.cid` is the like's "content id", i.e. a hash of its contents.  This is different from the DID of the author of the like.